src/middlewared/middlewared/etc_files/scst.conf.mako

```diff
@@ -67,35 +84,31 @@
                 if 'address' in addr:
                     failover_virtual_aliases.append(addr['address'])
 
+    cluster_mode_targets = []
+    cluster_mode_luns = {}
     if failover_status == "MASTER":
         middleware.call_sync("iscsi.target.logout_ha_targets")
         local_ip = middleware.call_sync("failover.local_ip")
         dlm_ready = middleware.call_sync("dlm.node_ready")
     elif failover_status == "BACKUP":
         if alua_enabled:
             logged_in_targets = middleware.call_sync("iscsi.target.login_ha_targets")
-            cluster_mode_targets = middleware.call_sync('failover.call_remote', 'iscsi.target.cluster_mode_targets')
+            try:
+                (cluster_mode_targets, cluster_mode_luns) = middleware.call_sync('failover.call_remote', 'iscsi.target.cluster_mode_targets_luns')
+            except CallError as e:
+                if e.errno != CallError.ENOMETHOD:
+                    raise
+                # We may have upgraded one node, but not yet the other
+                middleware.logger.warning('Failed to configure remote cluster_mode_targets_luns')
         else:
             middleware.call_sync("iscsi.target.logout_ha_targets")
+            targets = []
+            extents = portals = initiators = {}
```

Comment:
Just a note here. These all point to the same immutable dictionary object. This means if a change is made to `extents`, it'll also be reflected in `portals` and `initiators`
```
>>> a = b = c = {}
>>> a['key'] = 'value'
>>> 
>>> a
{'key': 'value'}
>>> b
{'key': 'value'}
>>> c
{'key': 'value'}
```
If this is expected, then ignore it 😄

Reply:
These values should not be modified after this point, but maybe it'll still be better to change just to avoid possible fragility.  Thanks.